.ddev/.gitignore

```diff
@@ -0,0 +1,41 @@
+#ddev-generated: Automatically generated ddev .gitignore.
+# You can remove the above line if you want to edit and maintain this file yourself.
+#/.gitignore
```

Comment:
I think if you manually make changes here you need to remove the first line. Otherwise, it will get re-added locally if you update ddev or install a project for the first time?

Reply:
I am not sure, I want to be informed about changes (you look at the diff before commiting, right?). If I remove the line, then ddev makes no adjustments anymore, which would not be desirable.

Reply:
Not sure I understand; if ddev updates the file, the gitignore will get ignored again, thus you won't see differences at all, because its no longer tracked.

Chicken/egg, I leave the decision up to you :-)

Reply:
Yup, but before committing, I am reviewing the changes, so see that it is set again, and can revert the change of this line.

Reply:
leave that line and that file ddev managed.

Do custom folder/file ignores either in the root level .gitignore or with a additional .gitignore in that folder (ensured to be added on global level).

If it is about the ddev gitignore as a startpoint - force add it. A added file will report changes even if it is in a ignore pattern included until it is removed from the repository.

Reply:
Then this might be the way, to force-add it. This change results from a discussion with Lina, I wdon't want to have a

    /.ddev
    !/.ddev/config.yaml

in the root's .gitignore file (as adding commands or configuration then has to be manually un-ignored in the root's .gitignore file)

Reply:
Ah true. I always forget about this method. Let's try this then :-)